content/docs/getting-started/adding-components/index.md

```diff
@@ -26,56 +38,36 @@ spec:
   lifecycle: production
 ```
 
-You can manually craft these YAML files and add them to Backstage using the catalog importer available at `/catalog-import`.
-
-Alternatively, Roadie can open a pull request into a repository to add the file.
-
-## Adding a component via automated pull request
-
-### Step 1. Open a PR
-
-Visit `https://your-company.roadie.so/catalog-import`.
-
-Enter the URL of a GitHub repository into the Repository URL input field and click Analyze. The next screen will allow you to customize the pull request before it is opened.
-
-![Catalog importer with URL in the input](./catalog-import-with-url.png)
+**Best Practice Tip!** </br>
+Ensure the name of the component will make sense to you and your team. This is how other users will refer to your component in Backstage.
 
-Ensure the Name of the created component will make sense to you and your team. This is how other users will refer to your component in Backstage.
+**Troubleshooting Note:**
+If you have not created YAML files you may see an option for Roadie to open a PR and create YAML files for you by opening a pull request into a repository to add the file. This feature is currently disabled as it requires too many permissions. We think we can built it better so it is under construction so we can build it with less permissions.
 
-![An input which allows setting the name of the component to be imported](./preparing-pull-request-name-component.png)
+### Step Two. Import Your Component YAML file into Backstage
+There are two ways you can get your components into Backstage:
+<ol>
+<li> You can manually add them to Backstage using the catalog importer available at `/catalog-import`, by copying and pasting the URL of the yaml file into the importer (see video below).</li>
+<li> Backstage can automatically discover them for you. Backstages scans your Github Org for new updates periodically. Once you have added a YAML file to any repo that you want to be imported to Backstage it will be found and autopopulated as a component.</li>
+</ol>
+If you are just getting started with Backstage we recomend adding your first component by method 1, as it can be faster and will get you going quicker. 
```

Comment:
```suggestion
If you are just getting started with Backstage we recommend adding your first component by method 1, as it can be faster and will get you going quicker. 

```